src/components/App/App.js

```diff
@@ -0,0 +1,20 @@
+import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Books from '../Books/Books';
+import Categories from '../Categories/Categories';
+import NavBar from '../NavBar/NavBar';
+
+const App = () => (
+  <div className="mymain">
+    <BrowserRouter>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/Books" element={<Books />} />
```

Comment:
- [ ] There is no need to implement repetitive routes here. The first path "/" should work just fine. Kindly remove the second one "/Books"